index.html

```diff
@@ -87,6 +87,14 @@
       <div class="medal">
         <a href='https://operationcode.org'><img src="images/medal_navy.png" alt="opcodelogo"></a>
       </div>
+      <ul class="supporters">
```

Comment:
Can you move this above the `Support` and below `Interested in learning to code?`